src/quick-view.scss

```diff
@@ -0,0 +1,93 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-quick-view;
+
+$fd-quick-view-avatar-indent: 0.75rem;
+$fd-quick-view-group-item-indent: 0.75rem;
+
+.#{$block} {
+  @include fd-reset();
+
+  @include fd-rtl() {
+    .#{$fd-namespace}-bar {
+      .#{$fd-namespace}-avatar {
+        margin-right: 0;
+        margin-left: $fd-quick-view-avatar-indent;
+      }
+    }
+  }
+
+  min-width: 20rem;
+  max-width: 30rem;
+
+  > *:first-child {
+    border-top-right-radius: $fd-popover-border-radius;
+    border-top-left-radius: $fd-popover-border-radius;
+  }
```

Comment:
I would go for `overflow: hidden` instead of this

Reply:
I thought about `overflow: hidden`, it's easier, but it could cause issues for inner content, for example for tooltips, or popups.
I think border-radius will be better in this case.